apis/v1alpha1/gateway_types.go

```diff
@@ -551,13 +551,19 @@ const (
 	// representation (see RFC 5952).
 	//
 	// Implementations should accept any address representation
-	// accepted by the inet_pton(3) API.
+	// accepted by the inet_pton(3) API. If the requested address
```

Comment:
we should probably lay this out from a different reference than linux manpages -- i noticed the bsd manpages have a slightly different description that doesn't include ipv6

Reply:
I'm not familiar with that RFC.
I can't seem to figure out what is it that we are trying to communicate here. Daneyon, could you give an example of what we mean by `any address representation`?